_source/_posts/2023-08-30-passwordless-authentication.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: blog_post
+title: "We're Living in a Passwordless World"
+author: alisa-duncan
+by: advocate
+communities: [security,mobile,javascript]
+description: "From passwords to passwordless — learn about how to elevate your authentication security."
+tags: ["passkey", "authentication", "fido"]
+tweets:
+- ""
+- ""
+- ""
+image: blog/passwordless-authentication/social.jpg
+type: awareness
+---
+
+Passwords? Bah! Who wants to remember all those letters, numbers, and special character combinations? Password managers and password manager browser extensions can help us, but what if we stopped using passwords instead? What if we stopped using other phishable authentication factors, such as SMS one-time passwords (OTP)? What if authentication was so seamless that we could use built-in device hardware capabilities instead?
```

Comment:
would it make sense to link "phishable" to a definition of phishing in this sentence, just to catch readers who aren't confident in their understanding of it?

Reply:
Good call, I'll add this.